### ISSUE TITLE
[Keyboard Navigation-Change time settings->Time Zone]: Scrolling functionality is not working while navigating list items under time zone dropdown.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66253]
### User Experience:  
Users who depend on keyboard for navigation will get disoriented and they will not be able to scroll content using keyboard and try accessing any element.

### Repro Steps:
 Open #Url: https://play.grafana.org/  Sign in with valid credentials. "Home" page appears. Navigate the page. Navigate the header controls "Apply time range", pane get expanded.  Navigate to the “Change time settings" button. Navigate the "Time zone" tab.  Verify and observe the issue.

### Actual Result: Issue: 
Scrolling functionality is not working while navigating list items under “time zone”dropdown.
Observation: While navigating time zone dropdown the list items is not scrolling down with the arrow keys and unable to see the remaining list.
 Expected Result:

### Solution: 
Scrolling functionality should work for ‘listitems’ under “Time zone “dropdown
Note: The same issue exists with the same controls throughout the application.

### Severity
`SEV2`

**Comment by ashharrison90** [comment]
@natellium i believe this is fixed already. i marked it fixed in [this spreadsheet](https://docs.google.com/spreadsheets/d/1kB-Sz3lvfje93HLVIx4UhDvwgU-J7k1PVN70ryQkc4M/edit#gid=1913685923). where are these issues coming from / are we cross-referencing with that spreadsheet? 🤔

**Comment by ashharrison90** [comment]
this is fixed already 👍
